src/slackify.js

```diff
@@ -53,7 +53,7 @@ const visitors = {
   url(node, text) {
     const url = this.encode(node.url || '', node);
     if (!isURL(url)) return url;
```

Comment:
I would suggest returning a raw url from this method, so it won't have `|` in the end of the url.
```suggestion
     if (!isURL(url) || !text) return url;
     return `<${url}|${text}>`;
```

Reply:
According to https://api.slack.com/docs/message-formatting#linking_to_urls
`URL detection is performed by the server. We do this so that URL detection (a non-trivial operation) is performed consistently across multiple clients.`
I guess it's better to send `<url>`

Reply:
Makes sense! 
So probably it should be something like this: 

```js

if (!isURL(url)) return url;
if (!text) return `<${url}>`
return `<${url}|${text}>`;
```
What do you think?

I like your original idea, but don't really like the `|` in the end 🙂

Reply:
Sure, No Problem!)